components/datetime/src/builder.rs

```diff
@@ -77,14 +77,14 @@
 //!
 //! // Time and Time Zone
 //! // Short length
-//! // Long specific non-location time zone
+//! // specific non-location time zone
 //! // Display time to the millisecond
 //! // Render for column alignment
 //!
 //! let static_field_set = fieldsets::T::short()
 //!     .with_time_precision(TimePrecision::FractionalSecond(FractionalSecondDigits::F3))
 //!     .with_alignment(Alignment::Column)
-//!     .with_zone_specific_long();
+//!     .with_zone(fieldsets::Z::short());
```

Comment:
Issue: It is important for the time zone name length to be selectable independently from the rest of the field set. I fixed this by making the field set identity define the time zone name length. An alternative design would be for the time zone name to carry a second length field, different than the overall skeleton length, but I ruled against this because the data is different enough. So the "Z = long specific, Zs = short specific, ..." model was the best overall design.

Reply:
This makes sense for the builder type